jest.config.js

```diff
@@ -1,10 +1,20 @@
+const path = require('path');
+const fs = require('fs');
+
+let envConfigPath = path.join(__dirname, 'fallback.env.config.js');
+const appEnvConfigPath = path.join(__dirname, 'env.config.js');
+
+if (fs.existsSync(appEnvConfigPath)) {
```

Comment:
Because you're putting the env.config.js mechanism directly into your app and can take for granted that the file exists, you shouldn't need any of this conditional logic or the fallback.env.config.js file - you know it's there.  This logic was only in frontend-build because most MFEs don't have an env.config.js file yet.

Reply:
simplified this file.